src/github.ts

```diff
@@ -2387,6 +2390,7 @@ export class GitHub {
           pull_number: pullRequestNumber,
           merge_method: mergeMethod,
         });
+        return 'direct-merged';
```

Comment:
Should the `.merge()` above be `await`ed?

Reply:
Of course, thanks for catching this one 🙀

Reply:
I added tests to cover that part, should be good now.